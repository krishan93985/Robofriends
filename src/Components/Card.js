import React from 'react';
const Card = ({ id,username,email}) => {
   // OR const { id,username,email} = props;
    return (
      <div className="tc grow bg-light-green dib br3 pa3 ma2 bw2 shadow-3">
          <img alt = 'icon' src={`https://robohash.org/${id}?size=200x200`} style={{display:"block",maxWidth:"100%"}} />
          <h2>{username}</h2>
          <p>{email}</p>
      </div>
    );
    }

export default Card;