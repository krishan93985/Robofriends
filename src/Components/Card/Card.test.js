import React from 'react';
import Card from './Card';
import { shallow, mount, render } from 'enzyme';
import toJson from 'enzyme-to-json';

it('is Testing Card component', () => {
    //shallowly renders single component
    //snapshot testing to check if component actually renders as expected
    const wrapper = shallow(<Card/>);
    expect(wrapper.length).toBe(1);
    expect(toJson(wrapper)).toMatchSnapshot();
})