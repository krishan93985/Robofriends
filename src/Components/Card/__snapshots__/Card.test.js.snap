// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`is Testing Card component 1`] = `
<div
  className="tc grow bg-light-green dib br3 pa3 ma2 bw2 shadow-3"
>
  <img
    alt="icon"
    src="https://robohash.org/undefined?size=200x200"
    style={
      Object {
        "display": "block",
        "maxWidth": "100%",
      }
    }
  />
  <h2 />
  <p />
</div>
`;
